features/user/managing_administrators/deleting_administrators.feature

```diff
@@ -0,0 +1,26 @@
+@managing_administrators
+Feature: Deleting an administrator
+    In order to delete administrator account
+    As an Administrator
+    I want to be able to delete the administrator
```

Comment:
delete the administaror? He won't be happy about that 💀 

`I want to be able to delete an other administrators account`?